### PR TITLE
Ross added t-Test to statslib

### PR DESCRIPTION
**Pull request by jrossthomson**
Please review.

### CODE REVIEW

**Comment by jrossthomson** [comment]
@danieldeleo Hi Dan, anything missing in this PR to move forward?  Thanks!

File: udfs/CONTRIBUTING.md [hunk 1]
```diff
@@ -58,6 +58,11 @@ the pull request can ensure a successful review process.
 Please follow these instructions to confirm that your test cases work as
 expected.
 
+> **WARNING:** Some UDFs require the use of Javascript libraries. The unit tests on 
+> these UDFs will fail during local testing if the JS libraries are not 
+> created locally. You can ignore these failures if you don't care about Javascript libraries. 
+> Or, **to avoid these failures**, follow the steps [here](js_libs/README.md#testing-js_libs-locally)
+
```
Comment (danieldeleo) [inline udfs/CONTRIBUTING.md:65]: Remove since this isn't the case anymore

File: udfs/cloudbuild_js_libs.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# Builds js_libs to your own GCS bucket for testing
```
Comment (danieldeleo) [inline udfs/cloudbuild_js_libs.yaml:1]: Remove this file

**Review by danieldeleo — CHANGES_REQUESTED** [review]

**Comment by danieldeleo** [comment]
Resolve conflicts as well

**Comment by danieldeleo** [comment]
Closing this since the changes are all repeated in PR#271

**Pull request closed without merging**